index.html

```diff
@@ -70,11 +70,11 @@
     <div class="modal-content">
       <div id="modal-title">Update Profile</div>
       <span class="closeUpdateProfile">&times;</span>
-      <img src="images/default-profile-pic.jpg" alt="Default">
-      <input type="text" id="updateUserFirstName" class="addContactFirstNameInput" value=firstName /><br />
-      <input type="text" id="updateUserLastName" class="addContactLastNameInput" value=lastName /><br />
-      <input type="text" id="updateUserPhone" class="phoneNumberInput" value=phone /><br />
-      <input type="text" id="updateUserEmail" class="emailInput" value=email /><br />
+      <input type="text" id="updateUserFirstName" class="updateFirstNameInput" value=firstName /><br />
+      <input type="text" id="updateUserLastName" class="updateContactLastNameInput" value=lastName /><br />
+      <input type="text" id="updateUserName" class="updateUserNameInput" value=userName /><br />
```

Comment:
Please remove this, I don't think we should allow people to change their usernames